.github/workflows/create-release.yml

```diff
@@ -66,7 +66,7 @@ jobs:
               
       - name: Update params.env with new release version
         run: |     
-          sed -i 's|:v[0-9.]*\b|:${{ github.event.inputs.tag_name }}|gm' config/overlays/odh/params.env
+          sed -i 's|:v[0-9.]*.*|:${{ github.event.inputs.tag_name }}|gm' config/overlays/odh/params.env

```

Comment:
Looking at `params.env`, it seems like that the  lines are in the format `name=image_name:fast`, which this regex will not match. Is there something else I'm missing?

Reply:
No , Yes, it doesn't start with v , On the release branches we will be having the tag starting with v but for the first release we will be having it has `fast` or `latest `so yesterday it was decided that for the tag we will be prefixing the version.

cc : @israel-hdez

Reply:
My understanding is that this new workflow would only be ran against `release-*` branches, not in `main` branch.
So, as part of our branch cut, we should modify `params.env` file to use `v{something}` tags. Ex: https://github.com/opendatahub-io/modelmesh-serving/blob/release-0.11.1/config/overlays/odh/params.env. Thus, the regex would match when the workflow is ran on `release-*` branches, which seems to be what we need.

Basically, what @rpancham said is right.